src/app/assessment/assessment.component.html

```diff
@@ -2,14 +2,16 @@
   <ion-toolbar color="light">
     <ion-icon name="arrow-back" (click)="back()" color="primary" slot="start" margin-start></ion-icon>
     <ion-title text-center>Assessment</ion-title>
+    <ion-title *ngIf="saving || doAssessment || doReview" text-center class="subTitle">{{savingMessage}}</ion-title>
```

Comment:
I think we don't need to put `saving` as the condition of this, it will always be displayed for `doAssessment` or `doReview`

Reply:
And actually we don't event need `saving` this variable at all

Reply:
yes we don't need saving variable.